core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -388,18 +385,17 @@ var ExplorationEditorPage = function() {
   // can then use to alter the state content, for example by calling
   // .appendBoldText(...).
   this.setContent = function(richTextInstructions) {
-    general.waitForSystem();
-    stateEditContent.isDisplayed().then(function() {
-      stateEditContent.click();
-    });
+    var until = protractor.ExpectedConditions;
```

Comment:
Hi @seanlip What do you think about this? I was having trouble with this element loading consistently so I wanted the browser to wait till the element is ready.

Reply:
I am a big fan of ExpectedConditions and I think, if anything, we should use it more! Well done on finding it; I think it is definitely better than stuff like general.waitForSystem().

And I think the syntax you use is fine. At first I thought declaring "until" as its own variable was a little weird, but it does read quite nicely with the browser.wait() call before it. So I'm all for it.